examples/react-native-vanilla/storybook/stories/Knobs/index.js

```diff
@@ -7,6 +7,7 @@ import {
   boolean,
   color,
   select,
+  radioButtons,
```

Comment:
This is now causing failures since `@storybook/addon-knobs/react` is deprecated and does not contain `radioButtons`?

Reply:
I imported radioButtons from @storybook/addon-knobs and removed the radio import from @storybook/addon-knobs/react.